source/index.md

```diff
@@ -32,6 +32,7 @@ maxdepth: 2
 caption: "Contents:"
 ---
 engineering/index
+mathematics/interactive
```

Comment:
Can do if you insist but it will not be very visible then and I thought this would be of interest for much wider audiences than Maths.

Reply:
I see your point, though I still think it would be better nested under `mathematics/` for now, as the article is fairly maths-focused.

I anticipate that most people will get to information in the community docs via direct links (e.g. when directed by a colleague or member of ACRC). I suspect the appearance of the index will not impact discoverability much!